must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -61,6 +64,13 @@ oc_yamls+=("volumesnapshot -A")
 oc_yamls+=("volumesnapshotclass")
 oc_yamls+=("volumesnapshotcontent")
 
+printf "collecting ocs-catalogsource pod logs from namspace %s \n" "${MARKETPLACE_NAMESPACE}"
+    LOG_OUTPUT=${BASE_COLLECTION_PATH}/namespaces/${MARKETPLACE_NAMESPACE}/oc_output/describe
+    mkdir -p "${LOG_OUTPUT}"
+    oc describe pod -n "${MARKETPLACE_NAMESPACE}" $(oc get pod -n openshift-marketplace --no-headers| grep catalog | awk '{print $1}') >> "${LOG_OUTPUT}"/gather-debug.log
```

Comment:
We should first check to see if the Pod even exists so we save ourselves a describe call that we know will fail.